src/Tools/ExternalAccess/FSharp/Internal/VisualStudio/FSharpProjectExternalErrorReporterFactory.cs

```diff
@@ -18,9 +18,7 @@ internal static class FSharpProjectExternalErrorReporterFactory
     {
         public static IVsLanguageServiceBuildErrorReporter2 Create(ProjectId projectId, string errorCodePrefix, IServiceProvider serviceProvider)
         {
-            ThreadHelper.ThrowIfNotOnUIThread();
```

Comment:
just to check, we're intentionally making this free threaded now?

Reply:
Correct: the only reason this had a UI thread affinity was to call SubscribeExternalErrorDiagnosticUpdateSourceToSolutionBuildEvents, but that's been moved earlier when the project itself is being created.